ssz-rs/src/de.rs

```diff
@@ -48,6 +51,7 @@ impl Display for DeserializeError {
             ),
             DeserializeError::InvalidInstance(err) => write!(f, "invalid instance: {err}"),
             DeserializeError::InvalidType(err) => write!(f, "invalid type: {err}"),
+            DeserializeError::InvalidRange{range, buffer_length} => write!(f, "failure on getting range: {range:#?}, for length: {buffer_length}"),
```

Comment:
if we just go w/ `ExpectedFurtherInput` we can drop all the changes to this

Reply:
Alright Il make the change and add some tests.